src/components/csvIntegration.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import { Segment, Button, Icon } from "semantic-ui-react";
+import { upperFirst } from "lodash"
+
+import { urlCsvIntegrtation } from '../urls';
+
+import style from "../styles.css";
+
+export class CsvIntegration extends Component {
```

Comment:
Rename it to https://github.com/IMGIITRoorkee/omniport-app-faculty-profile/pull/13#discussion_r454955633